packages/frontend/src/components/common/Dashboard/DashboardHeader.tsx

```diff
@@ -374,11 +375,7 @@ const DashboardHeader = ({
                         >
                             <ActionIcon
                                 variant="default"
-                                onClick={() => {
-                                    history.replace(
-                                        `/projects/${projectUuid}/dashboards/${dashboardUuid}/edit`,
-                                    );
-                                }}
+                                onClick={onEditClicked}
```

Comment:
so the `resetDashboardFilters` will do this `history` replace for us?

Reply:
`handleEnterEditMode` calls `resetDashboardFilters`, then does the redirect